.github/workflows/check_generated_code_drift.yml

```diff
@@ -0,0 +1,61 @@
+name: check_generated_code_drift
+on:
+  push:
+    branches:
+      - main
+    paths:
```

Comment:
We should drop the `paths` filter as it prevents us from adding this check as a required status check.
We use the `paths-filter` action as a solution (see below in this workflow).

This is explained a bit more in https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks (and suggests a different solution)

Reply:
Thanks! Updated, please take another look :Bow: